pages/common/git-log.md

```diff
@@ -14,6 +14,10 @@
 
 `git log --oneline`
 
+- Show the graph of commits in the current branch:
```

Comment:
Perhaps this will read better?

```suggestion
- Show a graph of commits in the current branch:
```

Reply:
It seems I don't have permission to push to this repo :(

Reply:
@mfrw: I don't think the OP has given us permission to write the the branch of this PR.